src/components/NotFoundPage/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+
+const NotFoundPage = () => (
+  <div className="container-fluid mt-3">
+    <Helmet>
```

Comment:
nit: to save on some lines, you can pass a `title` prop to the `<Helmet>` component.